### ISSUE TITLE
Add support to ArrayBuffer in idlharness

### ISSUE THREAD

**Issue by Ms2ger** [web-platform-tests/wpt#12850]
```
interface FileReader: EventTarget {
  readonly attribute (DOMString or ArrayBuffer)? result;
```

needs

```js
          idl_array.add_untested_idls(
            "[Exposed=(Window,Worker)] interface ArrayBuffer {};"
          );
```

to avoid an "Unrecognized type ArrayBuffer" `IdlHarnessError` from `IdlArray.prototype.assert_type_is`, which seems unhelpful.

**Comment by gsnedders** [comment]
Should we not just make sure we have all the JS built-ins?

**Comment by stephenmcgruer** [comment]
This was fixed in c46ec78846335260f32954bcafa1676d600836f0, I believe.
